Common/include/CConfig.hpp

```diff
@@ -3993,6 +3997,20 @@ class CConfig {
    */
   su2double GetRelaxation_DataDriven(void) const { return DataDriven_Relaxation_Factor; }
 
+  /*!
+   * \brief Get initial density value for Newton solver in data-driven fluid model.
+   * \return Initial density value.
+   */
+  su2double GetInitialDensity_DataDriven(void) const { return DataDriven_rho_init; }
+
+  /*!
+   * \brief Get initial static value for Newton solver in data-driven fluid model.
+   * \return Initial static value.
+   */
+  su2double GetInitialEnergy_DataDriven(void) const { return DataDriven_e_init; }
```

Comment:
Please make a dedicated struct to store the _DataDriven config options and replace all _DataDriven accessors which are becoming numerous.

Reply:
I've replaced the accessors for the CDataDrivenFluid fluid model now with structs. I did a similar thing for the CFluidFlamelet fluid model.